Thirty/Thirty/View Controllers/Root/RootViewController.swift

```diff
@@ -29,16 +29,13 @@ class RootViewController: UIViewController, UINavigationControllerDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .thPrimaryPurple
-        
+    }
+    
+    private func setupRootNavigationController() {
```

Comment:
ahh, you created this method which sets up the whole app's UI foundation - but then never called it.  I was so confused why nothing was loading.  I was cursing Firebase.  I fixed it in an upcoming PR.  But if you launch the app, know that's why nothing is showing up.